tests/utils/test_archive_scientific_readers.py

```
@@ -216,6 +216,37 @@ def test_read_7z_file_not_installed(self, tmp_path: Path) -> None:
+    def test_read_7z_file_with_mock_archive(self, tmp_path: Path) -> None:
+        """Cover archives.py lines 116-117,140-141 via a mocked py7zr."""
+        from types import SimpleNamespace
+        from unittest.mock import MagicMock, patch
+
+        import file_organizer.utils.readers.archives as _arc
+
+        fake_file = SimpleNamespace(filename="doc.txt", compressed=256, uncompressed=512)
+        fake_archive = MagicMock()
+        fake_archive.list.return_value = [fake_file]
+        fake_archive.password_protected = False
+        fake_archive.__enter__ = lambda s: fake_archive
```

Ruff (0.15.9)

[warning] Unused lambda argument: `s`

(ARG005)